### PR TITLE
Add coverm tools

### PR DESCRIPTION
**Pull request by gallardoalba**

### CODE REVIEW

**Comment by gallardoalba** [comment]
This is the final required step for including Coverm in usegalaxy.eu @mira-miracoli

**Review by bgruening — APPROVED** [review]

File: tools_iuc.yaml [hunk 1]
```diff
@@ -1599,6 +1599,14 @@ tools:
     owner: iuc
     tool_panel_section_label: Mapping
 
+  - name: coverm_contig
+    owner: iuc
+    tool_panel_section_label: Mapping
```
Comment (bebatut) [inline tools_iuc.yaml:1604]: I am not it is the best section for CoverM. I would have put it in "Metagenomic Analysis"
In any case, thanks a lot for adding it. I will tell Magnus and Valerie that the tool is there

Comment (gallardoalba) [reply tools_iuc.yaml:1604]: You are right, sorry; I didn't check the CoverM README.

Comment (bgruening) [reply tools_iuc.yaml:1604]: Can you change the category here ... I will fix it on the server.

Comment (gallardoalba) [reply tools_iuc.yaml:1604]: Sure.

**Pull request merged**